pkg/ui/workspaces/db-console/src/views/cluster/containers/dataDistribution/index.tsx

```diff
@@ -182,6 +182,15 @@ export class DataDistributionPage extends React.Component<
         <BackToAdvanceDebug history={this.props.history} />
         <section className="section">
           <h1 className="base-heading">Data Distribution</h1>
+          <p style={{ maxWidth: 300, paddingTop: 10 }}>
```

Comment:
Is there a tracking issue to fix the data distribution screen or remove it outright?

Reply:
There was https://github.com/cockroachdb/cockroach/issues/97942, but the Obs-Inf folks closed it. I'll defer to them if they want something to fix this and/or removing it outright.